src/ledger/NetworkConfig.cpp

```diff
@@ -578,35 +578,50 @@ updateCpuCostParamsEntryForV25(AbstractLedgerTxn& ltxRoot)
         switch (val)
         {
         case Bn254EncodeFp:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 345, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 344, 0);
             break;
         case Bn254DecodeFp:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 433, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 476, 0);
             break;
         case Bn254G1CheckPointOnCurve:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 858, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 904, 0);
             break;
         case Bn254G2CheckPointOnCurve:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 2642, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 2811, 0);
             break;
         case Bn254G2CheckPointInSubgroup:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 2930793, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 2937755, 0);
             break;
         case Bn254G1ProjectiveToAffine:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 63, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 61, 0);
             break;
         case Bn254G1Add:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 3465, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 3623, 0);
             break;
         case Bn254G1Mul:
-            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 1034667, 0);
+            params[val] = ContractCostParamEntry(ExtensionPoint{0}, 1150435, 0);
             break;
         case Bn254Pairing:
             params[val] =
-                ContractCostParamEntry(ExtensionPoint{0}, 5263357, 391912693);
+                ContractCostParamEntry(ExtensionPoint{0}, 10117409, 609283036);
```

Comment:
I asked a question about this in https://github.com/stellar/rs-soroban-env/pull/1608#discussion_r2492401733

Reply:
I replied there.